pkg/lib/verifier/rules/pcr_eventlog_includes.go

```diff
@@ -5,32 +5,46 @@
 package rules
 
 import (
-	"github.com/intel-secl/intel-secl/v3/pkg/hvs/constants/verifier-rules-and-faults"
+	"fmt"
+
+	constants "github.com/intel-secl/intel-secl/v3/pkg/hvs/constants/verifier-rules-and-faults"
 	"github.com/intel-secl/intel-secl/v3/pkg/lib/flavor/common"
 	"github.com/intel-secl/intel-secl/v3/pkg/lib/host-connector/types"
 	"github.com/intel-secl/intel-secl/v3/pkg/model/hvs"
 	"github.com/pkg/errors"
 )
 
-func NewPcrEventLogIncludes(expectedEventLogEntry *types.EventLogEntry, expectedPcr *types.Pcr, marker common.FlavorPart) (Rule, error) {
-	if expectedEventLogEntry == nil {
-		return nil, errors.New("The expected event log cannot be nil")
-	}
+//NewPcrEventLogIncludes creates the rule that will check
+//if all the actual event log measurements included in expected
+func NewPcrEventLogIncludes(expectedEventLogEntry *types.EventLogEntry, expectedPcrEventLogEntry *types.EventLogEntryFC, expectedPcr *types.Pcr, marker common.FlavorPart) (Rule, error) {
```

Comment:
I think we should have dual logic for the same thing,
expectedEventLogEntry and expectedPcrEventLogEntry 

@kwtj43 Let me know your views here. I think we must not have dual logic and dual structures defined for the same thing.